common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -281,21 +306,21 @@ innovation_hexer = {
 	flag = global_maa
 } 
```

Comment:
Ogres don't have Large Soldiers.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/155877987-5478ac66-14e5-40a1-9703-3db27c9a3ff0.png)

</details>